cores/arduino/stm32/usb/usbd_conf.c

```diff
@@ -520,9 +522,9 @@ USBD_StatusTypeDef USBD_LL_Init(USBD_HandleTypeDef *pdev)
 #else
   HAL_PCDEx_PMAConfig(&g_hpcd, 0x00, PCD_SNG_BUF, PMA_EP0_OUT_ADDR);
   HAL_PCDEx_PMAConfig(&g_hpcd, 0x80, PCD_SNG_BUF, PMA_EP0_IN_ADDR);
-  HAL_PCDEx_PMAConfig(&g_hpcd, 0x01, PCD_SNG_BUF, PMA_CDC_OUT_ADDR);
-  HAL_PCDEx_PMAConfig(&g_hpcd, 0x81, PCD_SNG_BUF, PMA_CDC_IN_ADDR);
-  HAL_PCDEx_PMAConfig(&g_hpcd, 0x82, PCD_SNG_BUF, PMA_CDC_CMD_ADDR);
+  HAL_PCDEx_PMAConfig(&g_hpcd, 0x01, PCD_DBL_BUF, PMA_CDC_OUT_ADDR);
```

Comment:
This change affect all families. Is there any risk?

Reply:
According to the reference manual it's impossible to use double buffering feature on both direction for one endpoint. If you use double buffering, you must chose one (IN or OUT) direction for the endpoint. 

I have been tested different endpoints for IN and OUT directions on BluePill and on Black F407VE board. And both work without problem. So, I think, that Switching IN and CMD endpoints to another numbers doesn't change anything. 

And HAL_PCDEx_PMAConfig used only for boards with USB (without OTG) feature. I think, that it only BluePill in supported variants list...